ios/AirMaps/AIRMapMarkerManager.m

```diff
@@ -33,7 +33,7 @@ - (UIView *)view
     return marker;
 }
 
-//RCT_EXPORT_VIEW_PROPERTY(identifier, NSString)
+RCT_EXPORT_VIEW_PROPERTY(identifier, NSString)
```

Comment:
I was playing around with this at some point but never really got it to work the way I wanted. I was hoping it could be more automatic.  Another thing is that we should make the public JS api accept a string or a number, rather than just a string (since numeric ids are common).